languages/elixir/exercises/concept/maps/.docs/introduction.md

```diff
@@ -0,0 +1,47 @@
+Maps in Elixir are the goto data structure for storing information in key-value pairs. In other languages, these might also be known as associative arrays (PHP), hashes (Perl 5, Raku), or dictionaries (Python).
+
+Keys and values can be of any data type, but if the key is an atom we can make use of special access protocols. Maps do not guarantee the order of their entries when accessed or returned.
```

Comment:
> we can make use of special access protocols

I, as a non-Elixir person, was slightly confused by what this means, especially the protocols bit. Maybe this can be phrased to something like:

```suggestion
Keys and values can be of any data type, but if the key is an atom we can use a shorthand syntax. Maps do not guarantee the order of their entries when accessed or returned.
```